test/unit/js/cssPrefix.test.js

```diff
@@ -37,23 +37,60 @@ test("When is not jQuery.cssHooks", function() {
     // When
 
     //Then
-    equal(method, false);
+    QUnit.equal(method, false);
```

Comment:
I thinkg it should be `assert.equal`
https://qunitjs.com/upgrade-guide-2.x/#replace-global-assertions-with-assert-arguments